server/app.js

```diff
@@ -28,7 +28,7 @@ const io = new Server(server, {
   },
 });
 
-require('./utils/socketServer')(io);
+//require('./utils/socketServer')(io);
```

Comment:
This is not needed anymore right?

Reply:
the commenting out isn't now that we merged the  socket

Reply:
Okay, got it 👍🏻